src/static/css/cf-enhancements.less

```diff
@@ -846,6 +843,338 @@ tbody {
+                Nov 4, 2013
+            </span>
+        </div>
+        <br>
+        <div class="meta-header">
+            <a href="#" class="meta-header_left category-slug">
+                <span class="cf-icon cf-icon-credit-card"></span>
+                Consumer finance
+            </a>
+        </div>
+      notes:
+        - "Just some random variations to show off what you could do."
+  tags:
+    - cf-typography
+*/
+
+.meta-header {
+    padding-bottom: unit(8px / @base-font-size-px, em);
+    margin-bottom: unit(30px / @base-font-size-px, em);
+    border-bottom: 1px solid @meta-header-border;
+}
+
+.meta-header_left {
+    margin-bottom: 0;
+}
+
+.meta-header_right {
+    display: block;
+    margin-bottom: 0;
+    .respond-to-min(@tablet-min, {
+        .u-right();
+        display: inline-block;
+    });
+}
+
+
+/* topdoc
+  name: Jump link
+  family: cf-typography
+  patterns:
+    - name: .jump-link
+      markup: |
+        <a class="jump-link" href="#">
+            <span class="jump-link_text">Default jump link</span>
+        </a>
+      codenotes:
+        - .jump-link
+    - name: .jump-link__large (modifier)
+      markup: |
+        <a class="jump-link jump-link__large" href="#">
+            <span class="jump-link_text">Large jump link</span>
+        </a>
+      codenotes:
+        - .jump-link__large
+      notes:
+        - "18px font-size, compared to the default of 16px"
+    - name: .jump-link__<icon> (modifier)
+      markup: |
+        <a class="jump-link jump-link__right" href="#">
+            <span class="jump-link_text">Jump link using the right icon from cf-icons</span>
+        </a>
+      codenotes:
+        - .jump-link__<icon>
+      notes:
+        - ".jump-link extends .icon-link, so all that is needed to add an icon
+           to a jump link is to add a modifier with the icon's name."
+    - name: .jump-link__before (modifier)
+      markup: |
+        <a class="jump-link jump-link__left jump-link__before" href="#">
+            <span class="jump-link_text">Jump link with icon on left</span>
+        </a>
+      codenotes:
+        - .jump-link__<icon>.jump-link__before
+      notes:
+        - "Jump links can also have icons before the text, like icon links."
+    - name: .jump-link__bg (modifier)
+      markup: |
+        <a class="jump-link jump-link__bg" href="#">
+            <span class="jump-link_text">Jump link with grey background and
+                solid borders on small screens</span>
+        </a>
+      codenotes:
+        - .jump-link__bg
+  tags:
+    - cf-typography
+*/
+
+.jump-link {
+    .webfont-medium();
+    font-size: unit(16px / @base-font-size-px, em);
+
+    .icon-link();
+
+    &__large {
+        font-size: unit(18px / @base-font-size-px, em);
+    }
+
+    &__underline {
+        border-bottom-width: 1px;
+    }
+
+    .respond-to-max(@mobile-max, {
+        .block-link();
+
+        &:after {
+            position: absolute;
```

Comment:
Adjust prop order here and throughout (maybe this should be a clean up task for the entire repo in the future?)